lib/mobility-core/src/Kernel/Types/Common.hs

```diff
@@ -125,10 +125,17 @@ data KVTable = KVTable
   }
   deriving (Generic, Eq, Show, ToJSON, FromJSON, FromDhall)
 
+--  drainer pseudocode:
+--  when enableKVForWriteAlso $ do
+--    if kafkaS3Tables
+--        then <push to kafka and then to s3>
+--        else <push to kafka as usually>
+
 data Tables = Tables
   { enableKVForWriteAlso :: [KVTable],
     enableKVForRead :: [Text],
-    kafkaNonKVTables :: [Text]
+    kafkaNonKVTables :: [Text],
+    kafkaS3Tables :: [Text] -- FIXME should work for non KV also
```

Comment:
Should we consider `kafkaNonKVTables` case?
I mean should we push to S3 tables, which is not in drainer? As per task description, we should not